easybuild/easyconfigs/l/libxml2/libxml2-2.9.4-foss-2016.04.eb

```diff
@@ -0,0 +1,29 @@
+easyblock = 'ConfigureMake'
```

Comment:
@AFCJamie please update to use the enhanced `libxml2` easyblock currently in `develop`, see https://github.com/hpcugent/easybuild-easyconfigs/pull/3509 (https://github.com/hpcugent/easybuild-easyblocks/pull/984)